aggregate_spatial_window.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "aggregate_spatial_window",
+    "summary": "Zonal statistics for rectangular windows",
+    "description": "Aggregates statistics over the spatial dimensions. This process passes a list of values to the reducer. \n\n The results are computed dividing the input X/Y grid into xDim/yDim non-overlapping windows and applying the reducer process to each window.\n If X and Y are not multiple of xDim/yDim NaNs padding before the computation is applied.",
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A raster data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            }
+        },
+        {
+            "name": "xDim",
```

Comment:
Parameter names in openEO are snake_case. Could we come up with a better name and description? For me it's not clear what this parameter excepts exactly. (Also applies for yDim.)

What's the behavior for null?

Reply:
Modified variable names to snake_case. x_size would be ok? It specifies the size of the window along the X dimension (or longitude). I also added the default to 1, making both x_size and y_size optional. So, the default window dimension would be 1x1 and it should produce the same result as apply.

Reply:
Let's check whether we can align with other processes regarding naming. 
Is this process only meant to work on horizontal spatial dimensions (x,y)?
Is 1x1 the most suitable default? It seems that's actually the most uncommon case as we have apply...

Reply:
Yes, it is meant to work on spatial dimensions (x,y).
I don't see a general default dimension, it should be the user to set it...we at Eurac would need it to apply a boxcar filtering operation on Sentinel-1 data, which could have dimensions 4x19 or 1x5 for example-